apps/etherpad/templates/services/nginx/configs/healthcheck.conf.j2

```diff
@@ -0,0 +1,10 @@
+server {
+  listen {{ etherpad_nginx_healthcheck_port }};
+  server_name localhost;
+
+  location {{ etherpad_nginx_healthcheck_endpoint }} {
+    add_header Content-Type text/plain;
+    access_log off;
+    return 200 "OK";
+  }
+}
```

Comment:
You should enable the nginx status page (see #479 )

Reply:
Good catch!